x-pack/plugins/apm/public/components/shared/charts/timeseries_chart.tsx

```diff
@@ -120,7 +133,24 @@ export function TimeseriesChart({
           showGridLines
         />
 
-        {showAnnotations && <Annotations />}
+        {showAnnotations && (
+          <LineAnnotation
+            id="annotations"
+            domainType={AnnotationDomainTypes.XDomain}
+            dataValues={annotations.map((annotation) => ({
+              dataValue: annotation['@timestamp'],
+              header: asAbsoluteDateTime(annotation['@timestamp']),
+              details: `${i18n.translate('xpack.apm.chart.annotation.version', {
+                defaultMessage: 'Version',
+              })} ${annotation.text}`,
+            }))}
+            style={{
+              line: { strokeWidth: 1, stroke: annotationColor, opacity: 1 },
+            }}
+            marker={<EuiIcon type="dot" color={annotationColor} />}
+            markerPosition={Position.Top}
+          />
```

Comment:
Any reason for this duplication? Wasn't it better to have it in a separate `Annotations` component?

Reply:
> Wasn't it better to have it in a separate <Annotations /> component?

Having it as a separate component is the reason the chart disappears after the annotations is rendered. It is a workaround until this issue gets fixed https://github.com/elastic/elastic-charts/issues/914